webapp/src/selectors.ts

```diff
@@ -205,3 +204,5 @@ export const isCloudTrialNeverStarted: (state: GlobalState) => boolean = createS
         return subscription?.trial_end_at === 0;
     },
 );
+
+export const channelState = (state: GlobalState, channelID: string) => getPluginState(state).channelState[channelID];
```

Comment:
I wonder if for this we should still offer a `isVoiceEnabled(state: GlobalState, channelID: string)` selector, so that it would simplify where you've used `channelState` above. (I'll make a comment there in a sec)

Reply:
Sure, I don't mind a further simplification as long as we pass the channelID.